cloudfront/provider.tf

```diff
@@ -0,0 +1,8 @@
+provider "aws" {
+  region  = "eu-west-1"
```

Comment:
I thought CloudFront distros had to live in us-east-1?

Reply:
They are "global" - certs for them must live in `us-east-1`.